core/src/main/scala/io/finch/metrics/Metrics.scala

```diff
@@ -0,0 +1,53 @@
+package io.finch.metrics
+
+trait Metrics {
+
+  /**
+    * Count a call to endpoint by path and method
+    */
+  def countCall(path: String): Unit
```

Comment:
How do I provide the HTTP Method to this method? Also, is this the materialized path or the matched path template/route?

Reply:
Good question about a method. Currently, there is no way to get endpoint's method. It's somewhere inside of the string. As I said, `Endpoint.Meta` would make it better.

It's a template path.

Reply:
Okay, I missed the `Endpoint.Meta` part in the PR description. :)